src/applications/caregivers/components/PreSubmitInfo/SignatureCheckbox.jsx

```diff
@@ -1,7 +1,8 @@
 import React, { useEffect, useState } from 'react';
 import PropTypes from 'prop-types';
 import ErrorableCheckbox from '@department-of-veterans-affairs/formation-react/ErrorableCheckbox';
-import SignatureInput from 'applications/caregivers/components/PreSubmitInfo/components/SignatureInput';
+
```

Comment:
I `un-nested` these files since they were just in a `dir` by themselves and test are in another dir so felt redundant